build.sbt

```diff
@@ -57,6 +57,9 @@ TwirlKeys.templateImports ++= Seq(
     "play.api.Play.current"
 )
 
+// include the enum path bindables
+routesImport += "model.editions._"
```

Comment:
ooh, tell me more!

Reply:
This enables you to use enums as bindables in routes: https://github.com/guardian/facia-tool/pull/999/files#diff-93c1ebbcb555c5f7d8ee08519974efa6R102 
PlayEnum generates them for free. Then you don't even need to check that the use has provided a sane value as the framework rejects anything that isn't with the appropriate 4xx error.